wiki/People/The_Team/Project_Loved_Team/en.md

```diff
@@ -51,7 +51,7 @@ The Project Loved Team mostly consists of *captains* for each [game mode](/wiki/
 
 Coordinators are mainly responsible for making sure all the captains work together to run Project Loved smoothly. They write news and forum posts, maintain tools, and help moderate discussions about Loved.
 
-- ![][flag_LT] [huu](https://osu.ppy.sh/users/6044237)
```

Comment:
you forgot to add huu to Retired members section in Project Loved Team page, specifically under Coordinators